release.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e
+
+[ $# == 1 ] || { echo "usage: $0 version" && exit 1; }
+
+VER=$1
+
+[[ "$VER" =~ ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$ ]] || {
+	echo "malformed version: \"$VER\""
+	exit 2
+}
+
+if test -n "$(git ls-files --others | \
+	grep --invert-match '\(vendor\|build/operator-sdk-v.\+\)')";
+then
+	echo "directory has untracked files"
+	exit 1
+fi
+
+if ! $(git diff-index --quiet HEAD --); then
+	echo "directory has uncommitted files"
+	exit 1
+fi
+
+# Update versions in code where necessary.
+CURR_VER="$(git describe --dirty --tags)"
+VER_FILE="${PWD}/version/version.go"
+TOML_TMPL_FILE="${PWD}/pkg/generator/templates.go"
+sed -i 's/\(Version = \)\"\(.\+\)\"/\1\"'${VER}'\"/' "$VER_FILE"
+sed -i 's/\(branch.\+\)\(\#osdk_branch_annotation\)/\# \1\2/' "$TOML_TMPL_FILE"
+sed -i 's/.\+\(\#osdk_version_annotation\)/  version = \"'${VER}'\" \1/' "$TOML_TMPL_FILE"
+
+git tag --sign --message "Operator SDK $VER" "$VER"
+
+git verify-tag --verbose "$VER"
+
+git commit -a -m "Version bump: '$CURR_VER' -> '$VER'"
```

Comment:
I think you're supposed to make the commit before the tag, otherwise the tagged commit will still have the old version number